Lib/ufo2ft/featureWriters/baseFeatureWriter.py

```diff
@@ -119,6 +153,110 @@ def _write(self):
         """Subclasses must override this."""
         raise NotImplementedError
 
+    def _insert(
+        self,
+        feaFile,
+        classDefs=None,
+        anchorDefs=None,
+        markClassDefs=None,
+        lookups=None,
+        features=None,
+    ):
+        """
+        Insert feature, its classDefs or markClassDefs and lookups at insert
+        marker comment.
+
+        If the insert marker is at the top of a feature block, the feature is
+        inserted before that block, and after if the insert marker is at the
+        bottom.
+        If the insert marker is in the middle of a feature block, that block
+        is split in to two blocks separated by the feature.
```

Comment:
this docstring is stale, the code below doesn't support insertion marker in the middle of a block

Reply:
Done.